code/src/components/ItemRemoved.js

```diff
@@ -0,0 +1,78 @@
+//ItemRemoved.js handles removal of todos/tasks
+
+import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+
+import Swal from "sweetalert2";
+//reference: https://sweetalert2.github.io/
+
+import styled from "styled-components";
+
+import { tasks } from "reducers/tasks";
+import { NumberofTasks } from "components/NumberofTasks";
+
+export const ItemRemoved = () => {
+  const dispatch = useDispatch();
+  const clearTasks = useSelector((store) => store.tasks.list.tasks.length);
+
+  const removeItems = () => {
+    Swal.fire({
+      title: "Do you want to remove all tasks?",
+      text: "This action cannot be undone once all tasks are deleted",
+      showCloseButton: true,
+      showCancelButton: true,
+      focusConfirm: false,
+      confirmButtonText: "yes, remove all",
+      confirmButtonColor: "#93dece",
+      confirmButtonAriaLabel: "yes",
+      cancelButtonText: "no, keep tasks",
+      cancelButtonColor: "#e18ea0",
+      cancelButtonAriaLabel: "no",
+    }).then((result) => {
+      if (result.isConfirmed) {
+        dispatch(tasks.actions.removeAllTasks());
+      }
+    });
+  };
+  return (
+    <>
+      <Button
+        type="button"
+        className="removeAll"
+        onClick={removeItems}
+        disabled={clearTasks === 0}
+      >
+        <h5>
+          remove all tasks
+          <span role="img" aria-label="bin">
+            🗑️
+          </span>
+        </h5>
+      </Button>
```

Comment:
You don't need the `h5` inside your button. Wouldn't be cool if this was not even rendered in the case of `clearTasks === 0` ? :)